packages/select-menu/src/index.tsx

```diff
@@ -183,6 +195,7 @@ class SelectMenuC<T extends OptionBase> extends React.Component<
                     <SelectMenuItem
                       appearance={this.props.appearance}
                       key={option.value}
+                      id={option.value}
```

Comment:
instead of passing `id`, you can approach it more efficiently: `<SelectMenuItem />` can return a proper `ref` only when it's actually selected. then, having this ref, you can scroll the menu to its `offsetTop` - and do it only upon popover being shown